pkg/install/install.go

```diff
@@ -310,7 +328,11 @@ func (o *Install) provisionEverestOperator(ctx context.Context, recVer *version.
 		v = recVer.EverestOperator.String()
 	}
 
-	if err := o.installOperator(ctx, everestOperatorChannel, common.EverestOperatorName, common.SystemNamespace, v)(); err != nil {
+	ch := everestBackendServiceName
```

Comment:
Should it be `everestOperatorChannel` instead?

Reply:
nice catch, updated.